base_tier_validation/models/tier_validation.py

```diff
@@ -35,11 +35,26 @@ class TierValidation(models.AbstractModel):
         search="_search_reviewer_ids",
     )
     can_review = fields.Boolean(compute="_compute_can_review")
+    has_comment = fields.Boolean(
+        related='review_ids.has_comment',
+    )
+    approve_level = fields.Boolean(
+        related='review_ids.approve_level',
+    )
 
     @api.multi
     def _compute_can_review(self):
         for rec in self:
             rec.can_review = self.env.user in rec.reviewer_ids
+            if rec.can_review and rec.approve_level:
+                sequence = rec.review_ids.filtered(
+                    lambda r: r.status in ('pending', 'rejected') and
+                    (self.env.user in r.reviewer_ids)).mapped('sequence')
+                my_sequence = sequence[0]
```

Comment:
* Can len(sequence) = 0, and will sequence[0] throw error?
* Can len(sequence) > 0, and why do we use index [0]. I.e., is it sorted somehow?

Reply:
`sequence` has at least 1 from `if rec.can_review and rec.approve_level:`